### ISSUE TITLE
[BUG] [TOGC PTR] Anub HC - Nerubian Burrower incorrect interaction with block

### ISSUE THREAD

**Issue by mpuryear** [JamminL/wotlk-classic-bugs#677]
#### 1.1. Description
  The Nerubian Burrowers apply an `Expose Weakness` that increases damage taken by a %. This % multiplier was always calculated **AFTER** block was applied, and this meant that block tanks were extremely effective on this encounter and it allowed guilds to only require 2 tanks. Currently the % modifier is being applied **BEFORE** block, which makes block much weaker than it used to be. The real headache that no other boss in WotLK requires 3 tanks and we are now expecting all raids to field a 3rd tank for an encounter that previously didn't require a third tank. Anecdotally, most players don't tend to enjoy pinch hitting as a role they don't main and as such will find this new interaction as unfun and disruptive.  
  
  The expected damage formula is `(DamageTaken - BlockValue) * exposeWeaknessModifier`
  The formula we are experiencing is `(DamageTaken * exposeWeaknessModifier) - BlockValue`
  #### 1.2. How to Reproduce
            -- Prereq: character that can block
           1. Spawn a Nerubian Burrower
           2. Let it stack Expose Weakness to its max of 10
           3. Notice that blocked hits are doing substantially more damage than expected
  #### 1.3  Source Material
  PTR Forum post with a few sources
https://us.forums.blizzard.com/en/wow/t/anub-hc-nerubian-burrower-incorrect-interaction-with-block/1582516

From a random log today 5/19. we can see that even a 7k block isn't enough to fully block the burrower hits, compared to the video in `2.2` where we see 3-4k being more than enough. 
https://classic.warcraftlogs.com/reports/fkbVcvKGm93AMQHa#fight=15&type=damage-taken&source=27&view=events&targetbuffs=67721
![image](https://github.com/JamminL/wotlk-classic-bugs/assets/17075809/77b3a301-b41b-4588-83f8-d3e96fa3610e)

#### 2.1. Description
The expected behavior is that we get `(DamageTaken - Block Value) * exposeWeaknessModifier`
  #### 2.2 Source Material
  There are examples in the PTR Forum post, but we can see in this video from 2009 that BV was definitely before the % modifier as we are seeing full blocks while the tank has 10 stacks. This is currently impossible on PTR. 
  https://www.youtube.com/watch?v=-EWFz94HQWI
  
  You can see that the damage is being fully blocked while at 10 stacks and the block amount is 3-4k. Thankfully they kept their entire scrolling combat log visible in the video. This wasn't changed until the 4.0 prepatch when block got reworked. 
  
  It's hard to tell because of video encoding at the time, but we can see in the video that ExposeWeakness is refreshed instead of applied and we immediately see full blocks 
  
![image](https://github.com/JamminL/wotlk-classic-bugs/assets/17075809/7ff1d313-1802-4283-b970-631f3d4dfa2e)

**Comment by Beannaa** [comment]
Further evidence that the current damage calculation for Expose Weakness on PTR is not accurate to the original game:
Patch 3.2.2 video: https://www.youtube.com/watch?v=lflL6DneEQA
Patch 3.3.3 Video: https://www.youtube.com/watch?v=Xp9o-zpaa5k
Patch 3.3.5 Thread: https://www.mmo-champion.com/threads/751841-Prot-Do-you-still-need-block-gear-for-Anub-25-men-HC

We haven't found any evidence that the damage calculation for Expose Weakness was ever changed throughout the course of the original WotLK expansion. Neither in the official patch notes, the "undocumented" changes recorded on the wiki, nor any of the major threads from EJ, TankSpot or any tank blog from the 2009-2010 era. It's hard to believe that, if solo-tanking Burrowers was made impossible overnight, none of these media would discuss it.
My anecdotal recollection of WotLK is that we used a single block tank all the way to Cata while we were still farming mounts. But memory is fallible which is why I looked for evidence that it may have been changed, without success.
